### ISSUE TITLE
Failed to allocate a colibri2 endpoint 

### ISSUE THREAD

**Issue by DoM1niC** [jitsi/jitsi-videobridge#1860]
*This Issue tracker is only for reporting bugs and tracking code related issues.*

Before posting, please make sure you check community.jitsi.org to see if the same or similar bugs have already been discussed. General questions, installation help, and feature requests can also be posted to community.jitsi.org.

## Description
Since last Jicofo changes / commits -> JVB doesn't work anymore 

### Log
```
cofo 2022-04-08 19:51:38.572 INFORMATION: [34] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24] ColibriV2SessionManager.allocate#251: Allocating for af357c0a
Jicofo 2022-04-08 19:51:38.572 INFORMATION: [36] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24] ColibriV2SessionManager.allocate#251: Allocating for 282ac779
Jicofo 2022-04-08 19:51:38.614 INFORMATION: [34] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24] ColibriV2SessionManager.allocate#276: Selected jvb1, session exists: false
Jicofo 2022-04-08 19:51:38.622 INFORMATION: [36] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24] ColibriV2SessionManager.allocate#276: Selected jvb1, session exists: true
Jicofo 2022-04-08 19:51:38.635 INFORMATION: [36] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24] ColibriV2SessionManager.handleResponse#357: Received error response: <iq xmlns='jabber:client' to='focus@auth.meet.3dns.eu/focus' from='jvbbrewery@conference.meet.3dns.eu/jvb1' id='V1SZE-11' type='error'><error type='cancel'><internal-server-error xmlns='urn:ietf:params:xml:ns:xmpp-stanzas'/><text xml:lang='en'>'boolean org.jitsi.xmpp.extensions.colibri2.Colibri2Endpoint.hasCapability(java.lang.String)'</text></error></iq>
Jicofo 2022-04-08 19:51:38.635 SCHWERWIEGEND: [36] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24] ColibriV2SessionManager.allocate#318: Failed to allocate a colibri2 endpoint for 282ac779
org.jitsi.jicofo.conference.colibri.BridgeFailedException:
        at org.jitsi.jicofo.conference.colibri.v2.ColibriV2SessionManager.handleResponse(ColibriV2SessionManager.kt:395)
        at org.jitsi.jicofo.conference.colibri.v2.ColibriV2SessionManager.allocate(ColibriV2SessionManager.kt:313)
        at org.jitsi.jicofo.conference.ParticipantInviteRunnable.doRun(ParticipantInviteRunnable.java:182)
        at org.jitsi.jicofo.conference.ParticipantInviteRunnable.run(ParticipantInviteRunnable.java:143)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)
Jicofo 2022-04-08 19:51:38.643 INFORMATION: [36] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24] ColibriV2SessionManager.removeParticipantInfosBySession#154: Removing session with no remaining participants: Colibri2Session[bridge=jvb1, id=b110278c-c39e-4367-af61-f12fad80ef83]
Jicofo 2022-04-08 19:51:38.647 WARNUNG: [34] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24] ColibriV2SessionManager.handleResponse#342: The session was removed, ignoring allocation response.
Jicofo 2022-04-08 19:51:38.647 SCHWERWIEGEND: [34] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24] ColibriV2SessionManager.allocate#318: Failed to allocate a colibri2 endpoint for af357c0a
org.jitsi.jicofo.conference.colibri.ColibriConferenceDisposedException:
        at org.jitsi.jicofo.conference.colibri.v2.ColibriV2SessionManager.handleResponse(ColibriV2SessionManager.kt:343)
        at org.jitsi.jicofo.conference.colibri.v2.ColibriV2SessionManager.allocate(ColibriV2SessionManager.kt:313)
        at org.jitsi.jicofo.conference.ParticipantInviteRunnable.doRun(ParticipantInviteRunnable.java:182)
        at org.jitsi.jicofo.conference.ParticipantInviteRunnable.run(ParticipantInviteRunnable.java:143)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)
Jicofo 2022-04-08 19:51:38.648 INFORMATION: [34] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24] ColibriV2SessionManager.removeParticipantInfosBySession#154: Removing session with no remaining participants: Colibri2Session[bridge=jvb1, id=b110278c-c39e-4367-af61-f12fad80ef83]
Jicofo 2022-04-08 19:51:38.648 SCHWERWIEGEND: [34] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24 participant=af357c0a] ParticipantInviteRunnable.doRun#201: Canceling due to
org.jitsi.jicofo.conference.colibri.ColibriConferenceDisposedException:
        at org.jitsi.jicofo.conference.colibri.v2.ColibriV2SessionManager.handleResponse(ColibriV2SessionManager.kt:343)
        at org.jitsi.jicofo.conference.colibri.v2.ColibriV2SessionManager.allocate(ColibriV2SessionManager.kt:313)
        at org.jitsi.jicofo.conference.ParticipantInviteRunnable.doRun(ParticipantInviteRunnable.java:182)
        at org.jitsi.jicofo.conference.ParticipantInviteRunnable.run(ParticipantInviteRunnable.java:143)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)
Jicofo 2022-04-08 19:51:38.649 SCHWERWIEGEND: [36] [room=generousvitaminsfloodeither@conference.meet.3dns.eu meeting_id=010fc130-342d-493e-af1e-f340ed5cba24 participant=282ac779] ParticipantInviteRunnable.doRun#223: Canceling due to
org.jitsi.jicofo.conference.colibri.BridgeFailedException:
        at org.jitsi.jicofo.conference.colibri.v2.ColibriV2SessionManager.handleResponse(ColibriV2SessionManager.kt:395)
        at org.jitsi.jicofo.conference.colibri.v2.ColibriV2SessionManager.allocate(ColibriV2SessionManager.kt:313)
        at org.jitsi.jicofo.conference.ParticipantInviteRunnable.doRun(ParticipantInviteRunnable.java:182)
        at org.jitsi.jicofo.conference.ParticipantInviteRunnable.run(ParticipantInviteRunnable.java:143)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)

```

## Expected Behavior
Commit before colibri v1 changes all is ok...

# Environment details
All Services / Web from master Branch self compiled

**Comment by damencho** [comment]
Can you also attach and the bridge logs for the problem?

**Comment by DoM1niC** [comment]
> Can you also attach and the bridge logs for the problem?

```
root@meet:~# =/opt/meet/videobridge/jvb.sh apis=rest
-bash: =/opt/meet/videobridge/jvb.sh: Datei oder Verzeichnis nicht gefunden
root@meet:~# /opt/meet/videobridge/jvb.sh apis=rest
JVB 2022-04-09 16:00:38.304 WARNUNG: [1] MainKt.main#61: A deprecated command line argument (--apis) is present. Please use the config file to control the REST API instead (see rest.md). Support for --apis will be removed in a future version.
JVB 2022-04-09 16:00:38.530 INFORMATION: [1] JitsiConfig.<clinit>#47: Initialized newConfig: merge of /opt/meet/videobridge/config/config.conf: 1,application.conf @ jar:file:/opt/meet/videobridge/jitsi-videobridge.jar!/application.conf: 1,system properties,reference.conf @ jar:file:/opt/meet/videobridge/jitsi-videobridge.jar!/reference.conf: 1,reference.conf @ jar:file:/opt/meet/videobridge/lib/ice4j-3.0-55-g32a8aad.jar!/reference.conf: 1,reference.conf @ jar:file:/opt/meet/videobridge/lib/jitsi-media-transform-1.0-292-g1c1e1b6.jar!/reference.conf: 1
JVB 2022-04-09 16:00:38.533 INFORMATION: [1] ReadOnlyConfigurationService.reloadConfiguration#51: loading config file at path /opt/meet/videobridge/config/sip-communicator.properties
JVB 2022-04-09 16:00:38.535 INFORMATION: [1] JitsiConfig.<clinit>#68: Initialized legacyConfig: sip communicator props (no description provided)
JVB 2022-04-09 16:00:38.535 INFORMATION: [1] JitsiConfig$Companion.reloadNewConfig#94: Reloading the Typesafe config source (previously reloaded 0 times).
JVB 2022-04-09 16:00:38.556 INFORMATION: [1] MainKt.main#89: Starting jitsi-videobridge version 2.1.SNAPSHOT
JVB 2022-04-09 16:00:38.918 INFORMATION: [12] org.ice4j.ice.harvest.MappingCandidateHarvesters.initialize: Adding a static mapping: StaticMapping(localAddress=10.0.0.6, publicAddress=80.147.149.90, localPort=null, publicPort=null, name=null)
JVB 2022-04-09 16:00:38.927 INFORMATION: [12] org.ice4j.ice.harvest.MappingCandidateHarvesters.initialize: Adding a static mapping: StaticMapping(localAddress=2002:dc3::6, publicAddress=2003:a:b15:4500::1, localPort=null, publicPort=null, name=null)
JVB 2022-04-09 16:00:38.945 INFORMATION: [12] org.ice4j.ice.harvest.MappingCandidateHarvesters.createStunHarvesters: Using 80.147.149.90:3478/udp for StunMappingCandidateHarvester (localAddress=10.0.0.6:0/udp).
JVB 2022-04-09 16:00:39.022 INFORMATION: [13] org.ice4j.ice.harvest.StunMappingCandidateHarvester.discover: Discovered public address 10.0.0.254:36612/udp from STUN server 80.147.149.90:3478/udp using local address org.ice4j.socket.IceUdpSocketWrapper@321cde18
JVB 2022-04-09 16:00:39.023 INFORMATION: [12] org.ice4j.ice.harvest.MappingCandidateHarvesters.initialize: Using org.ice4j.ice.harvest.StaticMappingCandidateHarvester(face=10.0.0.6:9/udp, mask=80.147.149.90:9/udp)
JVB 2022-04-09 16:00:39.024 INFORMATION: [12] org.ice4j.ice.harvest.MappingCandidateHarvesters.initialize: Using org.ice4j.ice.harvest.StaticMappingCandidateHarvester(face=[2002:dc3:0:0:0:0:0:6]:9/udp, mask=[2003:a:b15:4500:0:0:0:1]:9/udp)
JVB 2022-04-09 16:00:39.024 INFORMATION: [12] org.ice4j.ice.harvest.MappingCandidateHarvesters.initialize: Using org.ice4j.ice.harvest.StunMappingCandidateHarvester@38159d91
JVB 2022-04-09 16:00:39.025 INFORMATION: [12] org.ice4j.ice.harvest.MappingCandidateHarvesters.initialize: Initialized mapping harvesters (delay=433ms).  stunDiscoveryFailed=false
JVB 2022-04-09 16:00:39.173 INFORMATION: [20] [hostname=127.0.0.1 id=xmpp-server-1] MucClient.initializeConnectAndJoin#273: Initializing a new MucClient for [ org.jitsi.xmpp.mucclient.MucClientConfiguration id=xmpp-server-1 domain=auth.meet.3dns.eu hostname=127.0.0.1 port=null username=jvb mucs=[JvbBrewery@conference.meet.3dns.eu] mucNickname=jvb1 disableCertificateVerification=true]
JVB 2022-04-09 16:00:39.194 WARNUNG: [20] MucClient.createXMPPTCPConnectionConfiguration#116: Disabling certificate verification!
JVB 2022-04-09 16:00:39.213 INFORMATION: [1] LastNReducer.<init>#65: LastNReducer with reductionScale: 0.75 recoverScale: 1.25 impactTime: PT1M minLastN: 1 maxEnforcedLastN: 40
JVB 2022-04-09 16:00:39.215 INFORMATION: [1] TaskPools.<clinit>#81: TaskPools detected 4 processors, creating the CPU pool with that many threads
JVB 2022-04-09 16:00:39.217 INFORMATION: [1] HealthChecker.start#118: Started with interval=10000, timeout=PT30S, maxDuration=PT3S, stickyFailures=false.
JVB 2022-04-09 16:00:39.240 INFORMATION: [1] UlimitCheck.printUlimits#115: Running with open files limit 1048576 (hard 1048576), thread limit 15550 (hard 15550).
JVB 2022-04-09 16:00:39.242 INFORMATION: [1] VideobridgeExpireThread.start#88: Starting with 60 second interval.
JVB 2022-04-09 16:00:39.273 INFORMATION: [1] MainKt.main#120: Not starting CallstatsService, disabled in configuration.
JVB 2022-04-09 16:00:39.276 INFORMATION: [1] MainKt.main#129: Starting public http server
JVB 2022-04-09 16:00:39.299 INFORMATION: [20] [hostname=127.0.0.1 id=xmpp-server-1] MucClient.initializeConnectAndJoin#332: Dispatching a thread to connect and login.
JVB 2022-04-09 16:00:39.328 INFORMATION: [20] [hostname=127.0.0.1 id=xmpp-server-1] MucClient$2.connected#305: Connected.
JVB 2022-04-09 16:00:39.328 INFORMATION: [20] [hostname=127.0.0.1 id=xmpp-server-1] MucClient.lambda$getConnectAndLoginCallable$7#628: Logging in.
JVB 2022-04-09 16:00:39.337 INFORMATION: [1] ColibriWebSocketService.<init>#40: Base URL: wss://meet.3dns.eu:443/colibri-ws/jvb1 Relay URL: wss://meet.3dns.eu:443/colibri-relay-ws/jvb1
JVB 2022-04-09 16:00:39.357 INFORMATION: [1] org.eclipse.jetty.util.log.Log.initialized: Logging initialized @1274ms to org.eclipse.jetty.util.log.JavaUtilLog
JVB 2022-04-09 16:00:39.431 INFORMATION: [20] [hostname=127.0.0.1 id=xmpp-server-1] MucClient$2.authenticated#311: Authenticated, b=false
JVB 2022-04-09 16:00:39.449 INFORMATION: [1] ColibriWebSocketService.registerServlet#79: Registering servlet with baseUrl = wss://meet.3dns.eu:443/colibri-ws/jvb1, relayUrl = wss://meet.3dns.eu:443/colibri-relay-ws/jvb1
JVB 2022-04-09 16:00:39.458 INFORMATION: [20] [hostname=127.0.0.1 id=xmpp-server-1] MucClient$MucWrapper.join#752: Joined MUC: jvbbrewery@conference.meet.3dns.eu
JVB 2022-04-09 16:00:39.477 INFORMATION: [1] org.eclipse.jetty.server.Server.doStart: jetty-9.4.44.v20210927; built: 2021-09-27T23:02:44.612Z; git: 8da83308eeca865e495e53ef315a249d63ba9332; jvm 11.0.14.1+1-post-Debian-1
JVB 2022-04-09 16:00:39.536 INFORMATION: [1] org.eclipse.jetty.server.handler.ContextHandler.doStart: Started o.e.j.s.ServletContextHandler@7e0b9178{/,null,AVAILABLE}
JVB 2022-04-09 16:00:39.548 INFORMATION: [1] org.eclipse.jetty.server.AbstractConnector.doStart: Started ServerConnector@30272916{HTTP/1.1, (http/1.1)}{0.0.0.0:9090}
JVB 2022-04-09 16:00:39.549 INFORMATION: [1] org.eclipse.jetty.server.Server.doStart: Started @1466ms
JVB 2022-04-09 16:00:39.550 INFORMATION: [1] MainKt.main#147: Starting private http server
JVB 2022-04-09 16:00:39.598 INFORMATION: [1] org.eclipse.jetty.server.Server.doStart: jetty-9.4.44.v20210927; built: 2021-09-27T23:02:44.612Z; git: 8da83308eeca865e495e53ef315a249d63ba9332; jvm 11.0.14.1+1-post-Debian-1
JVB 2022-04-09 16:00:39.896 WARNUNG: [1] org.glassfish.jersey.server.wadl.WadlFeature.configure: JAXBContext implementation could not be found. WADL feature is disabled.
JVB 2022-04-09 16:00:39.999 WARNUNG: [1] org.glassfish.jersey.internal.inject.Providers.checkProviderRuntime: A provider org.jitsi.rest.Health registered in SERVER runtime does not implement any provider interfaces applicable in the SERVER runtime. Due to constraint configuration problems the provider org.jitsi.rest.Health will be ignored.
JVB 2022-04-09 16:00:40.278 INFORMATION: [1] org.eclipse.jetty.server.handler.ContextHandler.doStart: Started o.e.j.s.ServletContextHandler@6e03db1f{/,null,AVAILABLE}
JVB 2022-04-09 16:00:40.280 INFORMATION: [1] org.eclipse.jetty.server.AbstractConnector.doStart: Started ServerConnector@46c00568{HTTP/1.1, (http/1.1)}{0.0.0.0:8080}
JVB 2022-04-09 16:00:40.281 INFORMATION: [1] org.eclipse.jetty.server.Server.doStart: Started @2198ms
JVB 2022-04-09 16:00:49.192 INFORMATION: [22] HealthChecker.run#171: Performed a successful health check in PT0.000649S. Sticky failure: false
JVB 2022-04-09 16:00:58.768 INFORMATION: [50] [confId=1059deafb9cbfcfe gid=-2 conf_name=gladteensblockwhere@conference.meet.3dns.eu] EndpointConnectionStatusMonitor.start#57: Starting connection status monitor
JVB 2022-04-09 16:00:58.769 INFORMATION: [50] Videobridge.createConference#304: create_conf, id=1059deafb9cbfcfe gid=-2 meetingId=f7c24095-b19c-442c-a788-26a64193849c
JVB 2022-04-09 16:00:58.773 WARNUNG: [51] [confId=1059deafb9cbfcfe gid=-2 conf_name=gladteensblockwhere@conference.meet.3dns.eu] Conference.lambda$new$0#314: Failed to handle colibri request:
java.lang.NoSuchMethodError: 'boolean org.jitsi.xmpp.extensions.colibri2.Colibri2Endpoint.hasCapability(java.lang.String)'
        at org.jitsi.videobridge.colibri2.Colibri2ConferenceHandler.handleColibri2Endpoint(Colibri2ConferenceHandler.kt:166)
        at org.jitsi.videobridge.colibri2.Colibri2ConferenceHandler.handleConferenceModifyIQ(Colibri2ConferenceHandler.kt:71)
        at org.jitsi.videobridge.Conference.lambda$new$0(Conference.java:291)
        at org.jitsi.utils.queue.PacketQueue$HandlerAdapter.handleItem(PacketQueue.java:416)
        at org.jitsi.utils.queue.AsyncQueueHandler$1.run(AsyncQueueHandler.java:136)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)
JVB 2022-04-09 16:00:58.781 WARNUNG: [51] [confId=1059deafb9cbfcfe gid=-2 conf_name=gladteensblockwhere@conference.meet.3dns.eu] Conference.lambda$new$0#314: Failed to handle colibri request:
java.lang.NoSuchMethodError: 'boolean org.jitsi.xmpp.extensions.colibri2.Colibri2Endpoint.hasCapability(java.lang.String)'
        at org.jitsi.videobridge.colibri2.Colibri2ConferenceHandler.handleColibri2Endpoint(Colibri2ConferenceHandler.kt:166)
        at org.jitsi.videobridge.colibri2.Colibri2ConferenceHandler.handleConferenceModifyIQ(Colibri2ConferenceHandler.kt:71)
        at org.jitsi.videobridge.Conference.lambda$new$0(Conference.java:291)
        at org.jitsi.utils.queue.PacketQueue$HandlerAdapter.handleItem(PacketQueue.java:416)
        at org.jitsi.utils.queue.AsyncQueueHandler$1.run(AsyncQueueHandler.java:136)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)
JVB 2022-04-09 16:00:58.786 INFORMATION: [51] [confId=1059deafb9cbfcfe gid=-2 conf_name=gladteensblockwhere@conference.meet.3dns.eu] Colibri2ConferenceHandler.handleConferenceModifyIQ#90: All endpoints and relays removed, expiring.
JVB 2022-04-09 16:00:58.786 INFORMATION: [51] [confId=1059deafb9cbfcfe gid=-2 conf_name=gladteensblockwhere@conference.meet.3dns.eu] Conference.expire#661: Expiring.
JVB 2022-04-09 16:00:58.791 INFORMATION: [51] [confId=1059deafb9cbfcfe gid=-2 conf_name=gladteensblockwhere@conference.meet.3dns.eu] EndpointConnectionStatusMonitor.stop#65: Stopped
JVB 2022-04-09 16:00:58.793 INFORMATION: [51] [confId=1059deafb9cbfcfe gid=-2 conf_name=gladteensblockwhere@conference.meet.3dns.eu] Conference.updateStatisticsOnExpire#728: expire_conf,duration=0,has_failed=false,has_partially_failed=false
JVB 2022-04-09 16:00:59.191 INFORMATION: [22] HealthChecker.run#171: Performed a successful health check in PT0.000013S. Sticky failure: false
```

**Comment by DoM1niC** [comment]
> ```
> JVB 2022-04-09 16:00:58.773 WARNUNG: [51] [confId=1059deafb9cbfcfe gid=-2 conf_name=gladteensblockwhere@conference.meet.3dns.eu] Conference.lambda$new$0#314: Failed to handle colibri request:
> java.lang.NoSuchMethodError: 'boolean org.jitsi.xmpp.extensions.colibri2.Colibri2Endpoint.hasCapability(java.lang.String)'
> ```
> 
> This looks like you updated jitsi-videobridge without the corresponding jitsi-xmpp-extensions update?

I used the master of this Branch, every compile I do a clean before compile...

**Comment by DoM1niC** [comment]
is now fixed for me sry 1.0-45-g209f259 of jitsi-xmpp-extensions wasn't added to my Server GIT Stuff my mistake Guys .... -> Closed but THX for the advice's.
